Resources/Prototypes/Entities/Structures/Power/apc.yml

```diff
@@ -143,6 +143,8 @@
     priority: 1
   - type: StaticPrice
     price: 500
+  - type: BatteryDrinkerSource # Parkstation IPCs
```

Comment:
remove this BatteryDrinkerSource stuff and just make it reuse ninja code, add BatteryDrainerComponent to ipc prototype and have its batteryUid updated with item slot insert and remove events